### PR TITLE
Adding Alpha API support to the image create/deprecate processes.

### PR DESCRIPTION
**Pull request by bkatyl**
- Adding Alpha API support to the image create/deprecate processes
- This PR does not update any dependency

### CODE REVIEW

File: daisy/compute/compute.go [hunk 1]
```diff
@@ -350,7 +364,7 @@ func (c *client) operationsWaitHelper(project, name string, getOperation operati
 
 		switch op.Status {
 		case "PENDING", "RUNNING":
-			time.Sleep(1 * time.Second)
+			time.Sleep(5 * time.Second)
```
Comment (zoran15) [inline daisy/compute/compute.go:367]: do we still need this? If I remember correctly, this was done to reduce the frequence of wait operations? With v.0.44.0 GCP go library, we might not need to do this.

File: daisy/step_deprecate_images.go [hunk 2]
```diff
@@ -51,7 +54,11 @@ func (d *DeprecateImages) validate(ctx context.Context, s *Step) DError {
 			return Errf("cannot deprecate image %q: project does not exist: %q", di.Image, di.Project)
 		}
 
-		if !strIn(di.DeprecationStatus.State, deprecationStates) {
+		// Verify State is one of the deprecated states.
+		// The Alpha check also requires the value to not be emptry string as in that case the GA API will be used.
+		if di.DeprecationStatusAlpha.State != "" && strIn(di.DeprecationStatusAlpha.State, deprecationStates) {
```
Comment (zoran15) [inline daisy/step_deprecate_images.go:59]: I forgot, can di.DeprecationStatusAlpha be null?

File: daisy/compute/compute.go (hunk 1 above)
Comment (bkatyl) [reply daisy/compute/compute.go]: No we don't. Updated the PR to revert it back to 1 second.

**Review by zoran15 — APPROVED** [review]

File: daisy/step_deprecate_images.go (hunk 2 above)
Comment (bkatyl) [reply daisy/step_deprecate_images.go:59]: The deprecationStates lists empty string as an allowed value, so requiring di.DeprecationStatusAlpha.State to not be empty string ensures that when it's not set in the Alpha API data structure, we use the GA data structure.

**Pull request merged**